content/en/docs/concepts/storage/persistent-volumes.md

```diff
@@ -546,6 +544,8 @@ Older versions of Kubernetes also supported the following in-tree PersistentVolu
   (**not available** starting v1.25)
 * `storageos` - StorageOS volume.
   (**not available** starting v1.25)
+* [`rbd`](/docs/concepts/storage/volumes/#rbd) - Rados Block Device (RBD) volume 
+  (**not available** starting v1.31)
```

Comment:
Alphabetical order please
```suggestion
* [`rbd`](/docs/concepts/storage/volumes/#rbd) - Rados Block Device (RBD) volume.
  (**not available** starting v1.31)
* `storageos` - StorageOS volume.
  (**not available** starting v1.25)
```

Reply:
@sftim thanks. I will update it after https://github.com/kubernetes/kubernetes/pull/124544 is merged.